ui/components/multichain/badge-status/index.scss

```diff
@@ -0,0 +1,25 @@
+.multichain-badge-status {
+  padding: 0;
+
+  &__badge {
+    height: 16px;
```

Comment:
Can we use `height={4} width={4}` instead of using CSS?

Reply:
Hmm..I am not sure if we can use the height prop directly, also it's gonna use 10px for the non active state. which is not a multiple of 4. So, we can't use it.